src/ansys/fluent/core/launcher/machinelist.py

```diff
@@ -0,0 +1,467 @@
+"""A module that provides machine list construction for distributed parallel
+environments, including queueing systems.
+
+Currently supports UGE, LSF, PBS and SLURM by parsing the contents of
+the PE_HOSTFILE, LSB_MCPU_HOSTS, PBS_NODEFILE and SLURM_JOB_NODELIST
+variables, respectively.
+"""
+import csv
+import os
+import subprocess
+
+from ansys.fluent.core.launcher.machines import Machine, MachineList
+
+
+def loadMachines(machineDict=None, hostInfo=None, ncores=None):
```

Comment:
So... the return content of this would be used to construct a `-cnf` string to pass to Fluent.